<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;从&lt;a href=&quot;http://www.biqugse.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;笔趣阁&lt;/span&gt;&lt;/a&gt;下载小说&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>下载</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="EntryNum">
                <property name="toolTip">
                 <string>此处输入链接</string>
                </property>
                <property name="statusTip">
                 <string/>
                </property>
                <property name="whatsThis">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入完整链接&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="accessibleName">
                 <string/>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>在此处输入链接</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="Run">
                <property name="toolTip">
                 <string>“开始下载”按钮</string>
                </property>
                <property name="text">
                 <string>开始下载</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QProgressBar" name="PBar">
                <property name="toolTip">
                 <string>进度条</string>
                </property>
                <property name="whatsThis">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;完成百分比&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="textDirection">
                 <enum>QProgressBar::TopToBottom</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="tryTimes">
                <property name="maximum">
                 <number>1024</number>
                </property>
                <property name="value">
                 <number>3</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="Saver">
                <property name="toolTip">
                 <string>“保存为txt”按钮</string>
                </property>
                <property name="text">
                 <string>保存文本</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextBrowser" name="Info">
            <property name="toolTip">
             <string>预览框</string>
            </property>
            <property name="whatsThis">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;信息输出&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>搜索</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="enterBookName">
            <property name="toolTip">
             <string>此处输入书名</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>在此处输入书名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="search">
            <property name="toolTip">
             <string>搜索按钮</string>
            </property>
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTreeWidget" name="showLinks">
          <property name="toolTip">
           <string>直接点击即可选择</string>
          </property>
          <column>
           <property name="text">
            <string>书名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>链接</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>EntryNum</tabstop>
  <tabstop>Run</tabstop>
  <tabstop>Saver</tabstop>
  <tabstop>Info</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>enterBookName</tabstop>
  <tabstop>search</tabstop>
  <tabstop>showLinks</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
